### DISCUSSION TITLE
Database formulas slow down app considerably

### DISCUSSION THREAD

**Discussion by stellarfinn** [Budibase/budibase#11524] [Help]
Is anyone else using database formulas and experiencing a slow down in load times? I've noticed that each additional database formula increases the load time of the app.

**Comment by melohagan** [comment]
Hey @stellarfinn 

Do you mean the loading of data tables that have formulas is taking longer? If you are using **Dynamic** formulas, then the value is calculated on read of each row, which depending on what the formula is could be quite a lot of processing.
